components/infobox/wikis/starcraft2/infobox_person_player_custom.lua

```diff
@@ -139,6 +141,8 @@ function CustomInjector:addCustomCells(widgets)
 		currentYearEarnings = '$' .. mw.language.new('en'):formatNum(currentYearEarnings)
 	end
 
+	local yearsActiveCaster = CustomPlayer._getActiveCasterYears()
```

Comment:
For players this is behind `_shouldQueryData`. Should this be that as well?

Reply:
yeah, adjusted it and also retested stuff